<!--%Tag(FULL)%-->
<launch>
	
	<param name="music_dir" value="/home/haikalpribadi/Workspace/ROS/radiophonic/music"/>
	<param name="bextract_args" value="-chroma" />
	<!--<param name="osc_port" value="" />-->
	
	<node pkg="audiogesture" type="music_extractor" name="MusicFeatureExtractor" output="screen"/>
	<node pkg="audiogesture" type="sample_listener" name="SampleListener" output="screen"/>
	<node pkg="audiogesture" type="collection_generator" name="CollectionGenerator" output="screen"/>
	
</launch>
<!--%EndTag(FULL)%-->
